src/main/webapp/synchronizer.js

```diff
@@ -13,7 +13,9 @@
 // limitations under the License.
 
 let videoSyncTimer;
-const TIME_INTERVAL = 1000;
+let lastTime;
+
+const TIME_INTERVAL = 100;
```

Comment:
can you rename it to TIME_INTERVAL_MS so we know this is milliseconds.

Reply:
Done!